examples/pylab_examples/scatter_masked.py

```diff
@@ -7,17 +7,17 @@
 
 N = 100
 r0 = 0.6
-x = 0.9*np.random.rand(N)
-y = 0.9*np.random.rand(N)
-area = np.pi*(10 * np.random.rand(N))**2  # 0 to 10 point radiuses
+x = 0.9 * np.random.rand(N)
+y = 0.9 * np.random.rand(N)
+area = np.pi * (10 * np.random.rand(N)) ** 2  # 0 to 10 point radiuses
 c = np.sqrt(area)
-r = np.sqrt(x*x + y*y)
+r = np.sqrt(x * x + y * y)
```

Comment:
I don't think we have to use hypot *every time*, especially in examples where the function may be somewhat unknown.